backend/coins/btc/handlers/handlers.go

```diff
@@ -189,14 +190,21 @@ func (handlers *Handlers) getTxInfoJSON(txInfo *accounts.TransactionData, detail
 		Unit:   amount.Unit,
 	}
 	var formattedTime *string
+	var deductedAmount FormattedAmount
 	if txInfo.Timestamp != nil {
 		t := txInfo.Timestamp.Format(time.RFC3339)
 		formattedTime = &t
 		amountAtTime = handlers.formatAmountAtTimeAsJSON(txInfo.Amount, txInfo.Timestamp)
+		if txInfo.Fee != nil {
+			deductedAmount = handlers.formatAmountAtTimeAsJSON(coin.SumAmounts(txInfo.Amount, *txInfo.Fee), txInfo.Timestamp)
+		}
 	} else if txInfo.CreatedTimestamp != nil {
 		t := txInfo.CreatedTimestamp.Format(time.RFC3339)
 		formattedTime = &t
 		amountAtTime = handlers.formatAmountAtTimeAsJSON(txInfo.Amount, txInfo.CreatedTimestamp)
+		if txInfo.Fee != nil {
```

Comment:
nit: instead of duplicating the code (and to improve also the existing duplicated one), you could define a timestamp var which gets a value depending on `txInfo.Timestamp`/`txInfo.CreatedTimestamp`, and use it to initialize the other depending vars

Reply:
Also another possible nit: I'm wondering if it makes sense to compute the deductedAmount only for `send` txs. In send-to-self the fee is not nil, so we compute the deducted amount, which is not used anywhere, also because it is technically wrong, as the only deducted value is the fee. Just thinking, no strong opinion here

Reply:
> nit: instead of duplicating the code (and to improve also the existing duplicated one), you could define a timestamp var which gets a value depending on `txInfo.Timestamp`/`txInfo.CreatedTimestamp`, and use it to initialize the other depending vars

Yeah I was wondering the same but I wasn't sure about modifying unrelated code. definitely agree with you and going for this change.



> Also another possible nit: I'm wondering if it makes sense to compute the deductedAmount only for `send` txs. In send-to-self the fee is not nil, so we compute the deducted amount, which is not used anywhere, also because it is technically wrong, as the only deducted value is the fee. Just thinking, no strong opinion here

I think this one also makes sense!